src/Hydration.js

```diff
@@ -3,49 +3,29 @@ class Hydration {
     this.hydrationData = hydrationData;
   }
 
-findAUser(id) {
-  return this.hydrationData.filter(aUser => aUser.userID === id)
-
-}
-
-calcAverageOunces(id) {
-  let totalOunces = 0
-  let perDayHydration = this.hydrationData.filter((data) => id === data.userID)
-  perDayHydration.forEach((hydration) => (totalOunces += hydration.numOunces));
-  let avgOunces = totalOunces/perDayHydration.length
-  return avgOunces;
-}
-
-findByDate(dateSelected) {
-return this.hydrationData.find(data => data.date === dateSelected).numOunces;
-}
-
-selectWeek(id) {
-  let user = this.findAUser(id)
-  return user.slice(-7);
-}
-
-findWeeklyOunces() {
-  this.selectWeek()
-  let displaySevenDays = []
-  this.hydrationData.forEach((day) => {
-  displaySevenDays.push(day.numOunces);
-  });
-
-  // this.hydrationData.filter((day) => day.numOunces)
-  // .forEach((day) => displaySevenDays.push(day.numOunces));
-  console.log('>>>>>>', displaySevenDays);
-  return displaySevenDays;
-}
+  findAUser(id) {
+    return this.hydrationData.filter(aUser => aUser.userID === id)
+  }
+  calcAverageOunces(id) {
+    let totalOunces = 0
+    let perDayHydration = this.hydrationData.filter((data) => id === data.userID)
+    perDayHydration.forEach((hydration) => (totalOunces += hydration.numOunces));
+    let avgOunces = Math.round(totalOunces/perDayHydration.length)
+    return avgOunces;
+  }
+  findByDate(dateSelected) {
+  return this.hydrationData.find(data => data.date === dateSelected).numOunces;
```

Comment:
I think we need to indent here. (This is really just a test/learning these in line comments, pretty neat!)